tests/integration/test_sns.py

```diff
@@ -1045,6 +1045,166 @@ def get_messages(queue_url):
 
         retry(get_messages, retries=5, sleep=1, queue_url=queue_url)
 
+    def test_publish_batch_messages_from_fifo_topic_to_fifo_queue(self):
+        topic_name = "topic-{}.fifo".format(short_uid())
+        queue_name = "queue-%s.fifo" % short_uid()
```

Comment:
nitpick: recently we try to not mix string formatting styles and use f-strings. However the current code base is still full of the old styles and needs a rework anyway